pkg/ebpf/events_pipeline.go

```diff
@@ -325,6 +334,30 @@ func parseContextFlags(flags uint32) trace.ContextFlags {
 	}
 }
 
+// Get the syscall name from its ID, taking into account architecture and 32bit/64bit modes
+func parseSyscallID(syscallID int, isCompat bool, compatTranslationMap map[events.ID]events.ID) (string, error) {
```

Comment:
I don't see a case here for event which will not have an originated syscall (for example uprobe events I believe would fit this category), and that it is fine for them not to have such an origin. In that case the return should be `"", nil"`.
I assume when the syscall id passed here is 0 that would be the case, but we have to make sure the bpf code sets it as such.